app/src/main/java/com/concordium/wallet/ui/auth/login/AuthLoginActivity.kt

```diff
@@ -80,6 +80,14 @@ class AuthLoginActivity : BaseActivity() {
                 showPasswordError(value)
             }
         })
+        viewModel.errorSeedLiveData.observe(this) { error ->
+            if (error == true) {
```

Comment:
```suggestion
            if (error) {
```

but probably better to rename to `hasError`.